glacier/src/actions/add-fields.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type AddFieldsAction = ReduxStandardAction<"ADD_FIELDS", {fields: Field[], dataSource: string}>;
```

Comment:
If each `field` has a `DataSource`, why does the payload _also_ need to contain one outside any fields?